wayk_proto/src/sm/client_channels/clipboard.rs

```diff
@@ -74,12 +79,14 @@ enum ClipboardState {
     Capabilities,
     Disabled,
     Enabled,
+    EnabledWithAutoFetch,
```

Comment:
Not a big deal, but maybe this should be named like this
```suggestion
    AutoFetch,
```